File: test/fixtures/output/java/asynchttp/jsonObj-multiline.java
AsyncHttpClient client = new DefaultAsyncHttpClient();
client.prepare("POST", "https://httpbin.org/anything")
  .setHeader("content-type", "application/json")
  .setBody("{\n  \"foo\": \"bar\"\n}")
  .execute()
  .toCompletableFuture()
  .thenAccept(System.out::println)
  .join();

client.close();

File: __tests__/__fixtures__/output/php/guzzle/jsonObj-multiline.php
<?php
require_once('vendor/autoload.php');

$client = new \GuzzleHttp\Client();

$response = $client->request('POST', 'https://httpbin.org/anything', [
  'body' => '{
  "foo": "bar"
}',
  'headers' => [
    'content-type' => 'application/json',
  ],
]);

echo $response->getBody();
